resources/version.txt

```diff
@@ -1 +1 @@
-v1.0.1
+v1.0.2
```

Comment:
```suggestion
v1.1.0
```

Sorry, this slipped past me before... This project adheres to [semver](https://semver.org/) versioning, meaning this change constitutes a bump to the minor version.